src/js/components/award/visualizations/AmountsChart.jsx

```diff
@@ -170,89 +170,26 @@ export default class AmountsChart extends React.Component {
                     }}>
 
                     <g transform="translate(0, 5)">
```

Comment:
@kevinli-work Would it be worthwhile to move the drawing of the bars themselves down to their individual components as well?